### ISSUE TITLE
As a poll creator, it is somewhat surprising to not see responses until you've voted yourself

### ISSUE THREAD

**Issue by turt2live** [vector-im/element-web#20118]

**Comment by MurzNN** [comment]
I think that this even must be an option on poll creation form, to hide or not to hide results before user makes a vote.

**Comment by turt2live** [comment]
That is the eventual plan for polls, but not the scope of this issue :)

**Comment by jakewb-b** [comment]
Agreed. For the current implementation of polls (i.e. most users can only see results when they have voted), the poll creator should be able to see results immediately but should also still be able to vote.

**Comment by pherwaniharsh** [comment]
I'm also not able to see the total number of interacted votes, let alone the selected votes for the options within a poll at all. Whether or not I actually vote in it myself or not.

**Comment by turt2live** [comment]
@pherwaniharsh that would be a different, new, issue.
